wrp/messages.go

```diff
@@ -357,3 +367,17 @@ func (msg *ServiceAlive) BeforeEncode() error {
 	msg.Type = ServiceAliveMessageType
 	return nil
 }
+
+func findEventStringSubMatch(s string) string {
+	var (
+		eventPattern = regexp.MustCompile(`^event:(?P<event>[^/]+)`)
```

Comment:
We don't want to recompile this regex for each call.  Since this is an immutable variable, this should be a local global variable.  I'll merge & make this fix.

Reply:
I'm not sure why the build fails, but I can't merge until that works.

Reply:
Remove the changes to Gopkg.lock as we want to stay pinned to release 8.